timelock-impl/src/main/java/com/palantir/atlasdb/timelock/auth/config/ClientPrivilegesConfiguration.java

```diff
@@ -41,9 +43,12 @@
 
     abstract Set<TimelockNamespace> namespaces();
 
+    @JsonProperty("namespace-suffixes")
+    abstract Optional<Set<String>> namespaceSuffixes();
```

Comment:
I guess unless you want to be able to know explicitly when this hasn't been set, it seems like specifying a default of `ImmutableSet.of()` rather than making the return type be an `Optional` would work a bit more cleanly.

Reply:
Also, should we ban the empty string from being configured as a suffix?